packages/quantic/force-app/main/default/lwc/quanticResultList/quanticResultList.js

```diff
@@ -141,7 +141,10 @@ export default class QuanticResultList extends LightningElement {
 
   get fields() {
     if (this.fieldsToInclude.trim() === '') return [];
-    return this.fieldsToInclude.split(',').map((field) => field.trim());
+    return this.fieldsToInclude
+      .split(',')
+      .map((field) => field.trim())
+      .filter((field) => field.length > 0);
   }
```

Comment:
The `if` is not necessary anymore.

```suggestion
get fields() {
    return this.fieldsToInclude
      .split(',')
      .map((field) => field.trim())
      .filter((field) => field.length > 0);
  }
```